src/ol3-layerswitcher.js

```diff
@@ -190,7 +190,7 @@ ol.control.LayerSwitcher.prototype.renderLayer_ = function(lyr, idx) {
         li.appendChild(input);
 
         label.htmlFor = lyrId;
-        label.innerHTML = lyrTitle;
+        label.innerHTML = '&nbsp' + lyrTitle;
```

Comment:
@kpurdon why is a non-breaking space needed, is this in case the lyrTitle is an empty string?